src/main/java/by/exadel/internship/entity/Internship.java

```diff
@@ -55,11 +53,17 @@
     @Column(name = "inship_registration_end_date")
     private LocalDate registrationEndDate;
 
-//    private List<FormFullDTO> formList;
-//    private List<UserDTO> techList;
-//    private List<UserDTO> adminList;
-//    private List<LocationDTO> countryList;
-//    private InternshipFormat InternshipFormat;
-//    private List<Technology> technologyList;
+    @Enumerated(EnumType.STRING)
+    @Column(name = "inship_format_name", nullable = false)
```

Comment:
Why `nullable = false`? I think when admin create internship it's possible that format can not be approved yet.

Reply:
I am going to text  Zanna today about it with other questions.

Reply:
done like nullable to Internship and for db